.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+<!-- If this PR adds a CEP, do not replicate information in the document. Give a brief summary and link to related issues and PRs. -->
+
+# Summary of Changes
+<!--- In one or more sentences, describe the PR you are submitting. -->
+
+
+# Related CEPs and Issues
+<!--- Reference any related CEPs or issues and describe how this PR contributes to or addresses the problem. -->
```

Comment:
Should we add more here encouraging (requiring?) folks ensure that at least one issue exists already? And then the template can include
```
Fixes # 
```

as a prompt to reference that/those issues.

Reply:
I like the Fixes prompt. We could also update the comment to specify that an issue must exist before a PR can be made. I think MOOSE does that, and has all conversation in the issue (someone that actually interacts with that community can fact check that). I think this fits in with the "should we have a contributing guidelines document at the org level" that we have in #2. 

Related -- Do we want to open the discussions feature for this org? With the decline (or at least I feel like its declining) of the google email group, this could be a more accessible way for folks to ask questions about Cyclus. I've used the feature to ask questions in other repos when it's not really an issue.